### ISSUE TITLE
Rootcoord unittest bug

### ISSUE THREAD

**Issue by XuanYang-cn** [milvus-io/milvus#6567]
``` go  
  root_coord_test.go:450: 
        	Error Trace:	root_coord_test.go:450
        	Error:      	Not equal: 
        	            	expected: []string{"testColl_426334889178562561_1", "testColl_426334889178562561_0"}
        	            	actual  : []string{"testColl_426334889178562561_0", "testColl_426334889178562561_1"}
```
https://github.com/milvus-io/milvus/pull/6551/checks?check_run_id=3074525757

**Comment by czs007** [comment]
any update on this? @XuanYang-cn

**Comment by XuanYang-cn** [comment]
> any update on this? @XuanYang-cn

It's fixed on #6551.
